'ImageAnnotator' only refers to a type, but is being used as a value here.

Trying out v3 (specifically 3.0.0-rc.22), it looks like the `ImageAnnotator` component isn't being properly exported from the package. Using it like this (based on the readme):

```tsx
import { Annotorious, ImageAnnotator } from "@annotorious/react";
import "@annotorious/react/annotorious-react.css";

export default function Index() {
  return (
    <div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.8" }}>
      <h1>Welcome to Remix</h1>
      <Annotorious>
        <ImageAnnotator>
          <img src="my-image.jpg" alt="Example" />
        </ImageAnnotator>
      </Annotorious>
    </div>
  );
}
```

Produces the following error:
```
pnpm tsc
app/routes/_index.tsx:9:10 - error TS2693: 'ImageAnnotator' only refers to a type, but is being used as a value here.

9         <ImageAnnotator>
           ~~~~~~~~~~~~~~
```

Here's a simple repro (based on a vanilla Remix starter):
https://github.com/jrr/remix-annotorious-issue

--- a/packages/annotorious-react/src/index.ts
+++ b/packages/annotorious-react/src/index.ts
@@ -60,7 +60,6 @@ export type {
   FragmentSelector,
   ImageAnnotator as AnnotoriousImageAnnotator,
   ImageAnnotation,
-  ImageAnnotator,
   ImageAnnotatorState,
   Polygon,
   PolygonGeometry,

